int8 motor_state
float32 tau
float32 limit_current
float32 spd
float32 pos
float32 kp
float32 kd
